.gitignore

```diff
@@ -6,3 +6,4 @@ rdoc/*
 Gemfile.lock
 api_keys.yml
 test/geocoder_test.sqlite
+.idea
```

Comment:
This shouldn't be here because it's not Geocoder-specific. Please add to your `~/.gitignore_global` instead.